pkg/cluster/describe.go

```diff
@@ -118,6 +118,10 @@ func PrintClusterDesctipion(connection *sdk.Connection, cluster *cmv1.Cluster) e
 		cluster.CreationTimestamp().Round(time.Second).Format(time.RFC3339Nano),
 		cluster.ExpirationTimestamp().Round(time.Second).Format(time.RFC3339Nano),
 	)
+	shard := cluster.ProvisionShard()
```

Comment:
I don't think this is going to work. ProvisionShards don't get populated by the OCM API as part of the cluster object, but need a separate request (i.e. if you do `ocm get /api/.../cluster/{id}` you do not get the provision shard, you need to query the `/cluster/{id}/provision_shard` endpoint explicitly)

There are two ways to approach this:
1. Do a separate client query in `ocm describe` - this is easier and can be done as part of this PR with minimal changes
2. Expose provision shard in the cluster object directly (only populating it given the correct auth) - this is harder and will require a re-deploy of OCM, but will allow the usage of `ocm list clusters --columns=provision_shard.hive_config.server` without additional changest to `ocm-cli`

Reply:
Thanks @vkareh.

Its seems odd to me that the ocm-cli describe and the API would return different information that what is in the cluster object but it seems that we are already doing this for subscription and the cluster author. I'm +1 to just adding another call [here](https://github.com/openshift-online/ocm-cli/blob/1f8511a2e9a6ee67c01d9a3c110614a94b0fbbdb/pkg/cluster/describe.go#L28) to query for the shard ID if the user authorized. This would be quick and allow us to move forward.

Reply:
@vkareh I'm going to give option 1 a try. Seems like the best path forward here